platform/android/src/native_map_view.cpp

```diff
@@ -171,110 +171,183 @@ void NativeMapView::deactivate() {
  */
 void NativeMapView::invalidate() {
     android::UniqueEnv _env = android::AttachEnv();
-    static auto onInvalidate = javaClass.GetMethod<void ()>(*_env, "onInvalidate");
+    static auto onInvalidate = javaClass.GetMethod<void()>(*_env, "onInvalidate");
     javaPeer->Call(*_env, onInvalidate);
 }
 
 /**
- * From mbgl::RendererBackend. Callback to java NativeMapView#onMapChanged(int).
+ * From mbgl::RendererBackend. Callback to java NativeMapView#onCameraWillChange().
  *
  * May be called from any thread
  */
-void NativeMapView::notifyMapChange(mbgl::MapChange change) {
```

Comment:
Please also remove the declaration here: https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/src/native_map_view.hpp#L67